webpack.config.js

```diff
@@ -2,69 +2,52 @@
 
 var path = require("path");
 var webpack = require("webpack");
+var HtmlWebpackPlugin = require("html-webpack-plugin");
 
 module.exports = {
   devtool: "source-map",
   entry: [
-    "webpack-hot-middleware/client",
     "babel-polyfill",
-    "./index"
+    "./src/index"
   ],
   output: {
     path: path.join(__dirname, "dist"),
     filename: "bundle.js",
-    publicPath: "/dist/"
+    publicPath: "./"
   },
   plugins: [
-    new webpack.HotModuleReplacementPlugin(),
-    new webpack.NoEmitOnErrorsPlugin()
+    new webpack.DefinePlugin({
+      "process.env": {
+        "NODE_ENV": JSON.stringify(process.env.NODE_ENV)
+      }
+    }),
+    new HtmlWebpackPlugin({
+      template: path.join(__dirname, "public/index.html")
+    })
   ],
+  resolve: {
+    alias: {
+      assets: path.join(__dirname, "assets")
+    }
+  },
   module: {
-    loaders: [{
-      test: /\.md$/,
-      loader: "html-loader!markdown-loader?gfm=false"
-    }, {
+    rules: [{
       test: /\.(js|jsx)$/,
       exclude: /node_modules/,
-      loader: "babel-loader",
-      query: {
-        plugins: [
-          [
-            "react-transform", {
-              transforms: [{
-                transform: "react-transform-hmr",
-                imports: ["react"],
-                locals: ["module"]
-              }, {
-                transform: "react-transform-catch-errors",
-                imports: ["react", "redbox-react"]
-              }]
-            }
-          ]
-        ]
-      },
-      exclude: /node_modules/,
-      include: __dirname
+      loader: "babel-loader"
     }, {
       test: /\.css$/,
-      loaders: ["style-loader", "raw-loader"],
+      loaders: ["style-loader", "css-loader"],
       include: __dirname
     }, {
-      test: /\.svg$/,
-      loader: "url-loader?limit=10000&mimetype=image/svg+xml",
-      include: path.join(__dirname, "assets")
-    }, {
-      test: /\.png$/,
-      loader: "url-loader?mimetype=image/png",
-      include: path.join(__dirname, "assets")
+      test: /\.md$/,
+      loader: "html-loader!markdown-loader?gfm=false"
     }, {
-      test: /\.gif$/,
-      loader: "url-loader?mimetype=image/gif",
-      include: path.join(__dirname, "assets")
+      test: /\.(png|jpe?g|gif)$/,
```

Comment:
Merged these loaders, not sure if the mimetype was doing a whole bunch. If so, I'll switch it back!